### PR TITLE
Dpp 1029 new opa rulebase

### PR DESCRIPTION
**Pull request by carlbm**
Acceptance tests fail 24 (baseline)

One table now seems to be a view, there is code in the drop statement to also drop the table if it exists, but going back may be a problem

### CODE REVIEW

**Comment by carlbm** [comment]
Updated with a merge from develop
Fixed the failing tests
Fixed the failing integration tests

File: src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Rulebase.Ilr.ddl.tables.sql [hunk 1]
```diff
@@ -0,0 +1,1537 @@
...
+
+if object_id('[Rulebase].[EFA_SFA_Cases]','u') is not null
+begin
+	drop table [Rulebase].[EFA_SFA_Cases]
+end
+GO
+
+create table [Rulebase].[EFA_SFA_Cases]
+(
+	[LearnRefNumber] varchar(12),
```
Comment (lazcool) [inline src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Rulebase.Ilr.ddl.tables.sql:17]: The tables created in this file (and elsewhere, at least src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Valid.ddl.tables.sql) have LearnRefNumber as varchar(12), but elsewhere in this pull request (e.g. OPA.Transient.Invalid.ddl.tables.sql and OPA.Transient.Input.ddl.tables.sql), LearnRefNumber is changed from varchar(12) to varchar(100). Is this by design?

File: src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Valid.ddl.tables.sql [hunk 2]
```diff
@@ -311,6 +370,74 @@ create table [Valid].[LearningDelivery]
 )
 GO
 
+CREATE INDEX IDX_FundModel ON Valid.LearningDelivery (FundModel)
```
Comment (lazcool) [inline src/OpaEarning/Build/Deploy/1718/sql/ddl/OPA.Transient.Valid.ddl.tables.sql:373]: Should we check if the index already exists?

**Review by lazcool — COMMENTED** [review]
This is a large pull request against an area I'm not familiar with! However, I've added a couple of comments.

**Comment by satya-sanikommu** [comment]
Testing completed. No regression issues. Ready to merge

**Pull request merged**